receiver/statsdreceiver/README.md

```diff
@@ -12,27 +12,46 @@ The following settings are required:
 
 - `endpoint` (default = `localhost:8125`): Address and port to listen on.
 
+
+The Following settings are optional:
+
+- `aggregation_interval: 70s`(default value is 60s): The aggregation time that the receiver aggregates the metrics
+
 Example:
 
 ```yaml
 receivers:
   statsd:
   statsd/2:
     endpoint: "localhost:8127"
+    aggregation_interval: 70s
 ```
 
 The full list of settings exposed for this receiver are documented [here](./config.go)
 with detailed sample configurations [here](./testdata/config.yaml).
 
 ## Aggregation
 
-Currently the `statsdreceiver` is not providing any aggregation. There are
-ideas such as the [Metrics Transform Processor
-Proposal](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/332)
-that intend to enable control over Metric aggregation in a processor.
-
-An alternative will be to implement some simple aggregation in this receiver.
-
+In this new revision. Aggregation is done in statsD receiver. The default aggregation interval is 60s. The receiver only aggregates the metrics with the same metric name, metric type, label keys and label values. After each aggregation interval, the receiver will send all metrics(after aggregation) in this aggregation interval to the following workflow.
```

Comment:
I don't think you need to say "In this new revision" in the README

Reply:
Also you're missing a space here:

```
 metrics(after aggregation) 
```

Reply:
Sure, will revise